tests/test_other.py

```diff
@@ -4480,6 +4480,22 @@ def test_syscall_without_filesystem(self):
 }''')
     run_process([PYTHON, EMCC, 'src.c', '-s', 'NO_FILESYSTEM=1'])
 
+  def test_dashS(self):
+    run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.c'), '-S'])
+    self.assertExists('hello_world.s')
+
+  def test_dashS_stdout(self):
+    stdout = run_process([PYTHON, EMCC, path_from_root('tests', 'hello_world.c'), '-S', '-o', '-'], stdout=PIPE).stdout
+    self.assertEqual(os.listdir('.'), [])
+    self.assertContained('hello_world.c', stdout)
```

Comment:
This is asserting that the filename is in the generated asm?
I'm trying to think of a less-fragile piece of output we could depend on, but I got nothing, so this is probably fine.

Reply:
The problem is that the output here could be `.s` format or bitcode (in the case of fastcomp).   Once we ditch fastcomp we could check for more stuff here maybe.